systemtest/src/test/java/io/strimzi/systemtest/rollingupdate/KafkaRollerST.java

```diff
@@ -72,19 +72,27 @@ void testKafkaRollsWhenTopicIsUnderReplicated() {
         String topicName = KafkaTopicUtils.generateRandomNameOfTopic();
         timeMeasuringSystem.setOperationID(timeMeasuringSystem.startTimeMeasuring(Operation.CLUSTER_RECOVERY));
 
-        KafkaResource.kafkaPersistent(CLUSTER_NAME, 4)
-            .editSpec()
+        // We need to start with 3 replicas / brokers,
+        // so that KafkaStreamsTopicStore topic gets set/distributed on this first 3 [0, 1, 2],
+        // since this topic has replication-factor 3 and minISR 2.
+        KafkaResource.kafkaPersistent(CLUSTER_NAME, 3)
+                .editSpec()
                 .editKafka()
-                    .addToConfig("auto.create.topics.enable", "false")
+                .addToConfig("auto.create.topics.enable", "false")
                 .endKafka()
-            .endSpec()
-            .done();
+                .endSpec()
+                .done();
```

Comment:
Can you keep the formatting we had before here? It was intentional.

Reply:
Ah, OK.